src/oss-ci-cd-tooling/orb.yml

```diff
@@ -174,7 +206,7 @@ commands:
       - run:
           name: Restoring Verdaccio storage
           command: |
-            mkdir -p "<< parameters.verdaccio_path >>"
+            mkdir -p "$(dirname << parameters.verdaccio_path >>)"
```

Comment:
Note: The bug: Failure to create the parent directory, rather than the directory to restore.